dev/infra_server_objects_count_collector.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
seems you missed the empty case 
<img width="1283" alt="image" src="https://user-images.githubusercontent.com/44021874/227972196-d810dc0d-4f72-4c69-afa9-b51c8fead1ec.png">